openflights.js

```diff
@@ -2286,10 +2291,21 @@ function listFlights(str, desc, id) {
   $("result").innerHTML = table.join("");
   // Refresh sortables code
   sortables_init();
+
+  // Great Circle Mapper button setup
+  document.getElementById("gcmapbutton").onclick = () => {
+    const gcmURL = new URL("http://www.gcmap.com/mapui");
+    gcmURL.searchParams.append(
+      "P",
+      gcmFlightList.map((f) => `${f.src_code}-${f.dst_code}`).join(",")
+    );
+    gcmURL.searchParams.append("MS", "bm"); // 'M'ap 'S'tyle: 'b'lue 'm'arble
+    location.assign(gcmURL.href);
```

Comment:
Can we open this in a new tab, to keep the same behaviour?

`window.open(gcmURL.href, "_blank");` works... Not sure if there's a nicer/better way offhand.

Reply:
Seems `window.open(url, "openflights_export");` is what we're already doing